nss-day-cohort-39/nutshell-zaffre-zorillas
# Registration and Login complete

### Changes
- added event listener to registration form to dispatch user information
- nutshell listens for when a user is created to render the home page
- added event listener to sign in form for when a user signs in
- event listener checks to make sure password is correct and then renders homepage
- used session storage to store user id

### testing
- serve to localhost 5000
- serve json to 3000
- make sure welcome message renders
- click register and fill out form 
- put debugger into RegistrationForm.js on line 65, complete form and register user
- ensure that user id is being stored in session storage and in the details of the custom event declared on line 57
-continue running code and ensure that homepage renders
- reload page and sign in as existing user
- test incorrect password to make sure user is alerted
-put debugger into SigninForm.js on line 48 and make sure user id is being stored in sesstion storage and in the details of the custom event 
- use correct password and ensure that homepage loads
